content/general/concepts/full-stack/full-stack.md

```diff
@@ -0,0 +1,78 @@
+---
+Title: 'Full-Stack'
+Description: 'Full-stack development refers to having competency in both client & server side software.'
+Subjects:
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Browsers'
+  - 'Debugging'
+  - 'Development'
+  - 'Frameworks'
+  - 'Responsive'
+  - 'Servers'
+CatalogContent:
+  - 'paths/front-end-engineer-career-path'
+  - 'paths/back-end-engineer-career-path'
```

Comment:
```suggestion
```

In general we only have two entries for catalog content, only one of which is a "path" entry. This edit matches the "back-end" and "front-end" entries.

Reply:
@SSwiniarski Thanks!